src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/Repository/DataSetVersionImportRepository.cs

```diff
@@ -0,0 +1,31 @@
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Database;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Repository.Interfaces;
+using Microsoft.EntityFrameworkCore;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Repository;
+
+public class DataSetVersionImportRepository(PublicDataDbContext publicDataDbContext) : IDataSetVersionImportRepository
```

Comment:
It feels like there are going to be a few places that set statuses / stages etc so this kept the code a bit more succinct.

I've removed them for now because we did say that, yep.